pkg/watchdog/synchronized.go

```diff
@@ -15,13 +15,13 @@ var _ Watchdog = &synchronizedWatchdog{}
 
 // synchronizedWatchdog implements the Watchdog interface with synchronized calls of the implementation specific methods
 type synchronizedWatchdog struct {
-	impl         watchdogImpl
-	timeout      time.Duration
-	isStarted    bool
-	stop         context.CancelFunc
-	mutex        sync.Mutex
-	lastFoodTime time.Time
-	log          logr.Logger
+	impl                   watchdogImpl
+	timeout                time.Duration
+	isStarted, isRebooting bool
```

Comment:
while this is possible of course, I think it's best practice to use a new line for each field?

Reply:
As a rule of thumb I think you are right, however in this case those variables are closely related so I think this grouping makes sense.